frontend_vue/src/components/ManageToken.vue

```diff
@@ -96,7 +96,7 @@ const error = ref(null);
 const manageTokenResponse = ref();
 const tokenLogoUrl = ref();
 
-const dynamicComponent = ref({
+const dynamicComponent = shallowRef({
```

Comment:
it's like a ref, but only reactive to changes on the object itself, not in the props. 
It should be a good practice for performances. 
In this case, the component is quite small so the difference should be minimal, but I think it's good to start leveraging more of the vue functionalities

In this case, it should work, as I'm always reassigning the component dynamicComponent.value = SomeComponent;
The reactivity system will pick up the change and update the component.

It would not work if I'd plan to mutate just the props 
dynamicComponent.value.props.something= 'blabla';

Reply:
Ok, but if the props change in the parent? this won't update? that sounds like a painful debugging experience for someone trying to figure that out? i'm also not sure we need to care too much about performance?

Reply:
ok then, if it makes it just more complex I'm fine removing it and using the regular ref 🌈
I update it🙏